framework/log/Dispatcher.php

```diff
@@ -113,7 +113,7 @@ public function init()
     public function getLogger()
     {
         if ($this->_logger === null) {
-            $this->setLogger(Yii::getLogger());
+            $this->setLogger('yii\log\Logger');
```

Comment:
2.1 is set for 7.1 if im not mistaken https://github.com/yiisoft/yii2/issues/11397